package/network/ipv6/6rd/files/6rd.sh

```diff
@@ -9,6 +9,11 @@
 	init_proto "$@"
 }
 
+ipcalc() {
```

Comment:
Let's put this into `/lib/functions.sh`.

Reply:
:+1:
Nice - give me a sec

Reply:
Like that?
You can just resolve if there's nothing left to do.